UnmaskedEngine/View Models/GameViewModel.swift

```diff
@@ -154,15 +187,65 @@ public class GameViewModel {
         hero1?.name ?? "Masked"
     }
     
+    public var heroOneID: String {
+        hero1?.id ?? "1"
+    }
+    
     public var heroTwoName: String {
         hero2?.name ?? "Masked"
     }
     
+    public var heroTwoID: String {
+        hero2?.id ?? "2"
+    }
+    
     public var currentScore: String {
-        "\(score)"
+        "Score: \(score)"
     }
     
     public var statName: String {
         stat.rawValue
     }
 }
+
+@available(iOS 9.0, *)
+extension GameViewModel: WCSessionDelegate {
+    
+    public func sessionDidBecomeInactive(_ session: WCSession) {
+    }
+    
+    public func sessionDidDeactivate(_ session: WCSession) {
+    }
+    
+    @available(iOS 9.3, *)
+    public func session(_ session: WCSession, activationDidCompleteWith activationState: WCSessionActivationState, error: Error?) {
+    }
+    
+    public func session(_ session: WCSession,
+                 didReceiveMessage message: [String : Any]) {
+        handleSession(session,
+                      didReceiveMessage: message)
+    }
+    
+    public func session(_ session: WCSession,
+                 didReceiveMessage message: [String : Any],
+                 replyHandler: @escaping ([String : Any]) -> Void) {
+        handleSession(session,
+                      didReceiveMessage: message,
+                      replyHandler: replyHandler)
+    }
+    
+    public func handleSession(_ session: WCSession,
+                       didReceiveMessage message: [String : Any],
+                       replyHandler: (([String : Any]) -> Void)? = nil) {
+        DispatchQueue.main.async {
+            if let isHeroOne = message["isHeroOne"] as? Bool {
+                if isHeroOne {
+                    self.heroButtonPressed(isHeroOne: isHeroOne)
```

Comment:
Hmm rename this function, little but strange to be referencing a button tap in the `ViewModel`